src/Persistence.Tests/MsApp/MsappArchiveTests.cs

```diff
@@ -64,28 +65,11 @@ public void AddEntryTests(string[] entries)
         msappArchive.CanonicalEntries.Count.Should().Be(entries.Length);
         foreach (var entry in entries)
         {
-            msappArchive.CanonicalEntries.ContainsKey(MsappArchive.NormalizePath(entry)).Should().BeTrue();
+            msappArchive.CanonicalEntries.ContainsKey(FileUtils.NormalizePath(entry)).Should().BeTrue();
         }
 
         // Get the required entry should throw if it doesn't exist
         var action = () => msappArchive.GetRequiredEntry("not-exist");
         action.Invoking(a => a()).Should().Throw<FileNotFoundException>();
     }
-
-    [TestMethod]
-    [DataRow(@"_TestData/AppsWithYaml/HelloWorld.msapp", 14, 2, "HelloScreen", "screen", 8)]
-    public void Msapp_ShouldHave_Screens(string testFile, int allEntriesCount, int controlsCount,
```

Comment:
Please do not delete unit tests

Reply:
I dropped the code that loads Screens from msapp, this testt is not necessary anymore. We have the same thing already in Docserver. Will eventually migrate that over here in time